src/App.tsx

```diff
@@ -28,15 +38,84 @@ const MapListScreen = () => {
   return <MapList></MapList>;
 };
 
+//Created dummy About Us screen to test all three tabs
+const AboutUsScreen = () => {
+  return (
+    <View
+      style={{
+        flex: 1,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#ffffff",
+      }}
+    >
+      <Text style={{ fontSize: 30 }}>About Us</Text>
+    </View>
+  );
+};
+
+// If the focused route is not found, we need to assume it's the initial screen
+// This can happen during if there hasn't been any navigation inside the screen
+// In our case, it's "Home" as that's the first screen inside the navigator
+const getHeaderTitle = (route) => {
+  const routeName = getFocusedRouteNameFromRoute(route) ?? "Home";
+
+  switch (routeName) {
+    case "Home":
+      return "Home";
+    case "Map":
+      return "MapList";
+    case "About Us":
+      return "About Us";
+  }
+};
+
+const Tabs = ({ navigation, route }) => {
```

Comment:
There's an issue with the typing on {navigation, route}. It would be good practice for you to solve it on your own, but if you need help, let me know.

Reply:
By writing "noImplicitAny": false in tsconfig.json file I'm rid of type error. But, not sure it is correct solution or not. Some people are commenting that it doesn't fix root cause. I'm new to TS, need your help

Reply:
Yeah, please don’t use that option, it breaks proper type checking. I’ll post an example of what to do later.